lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -711,6 +711,20 @@ def test_downsampling_over_subset():
     )
 
 
+def test_downsampling_out_of_range():
+    num_points = 4
+    s = channel.Slice(channel.Continuous(np.arange(num_points), with_offset(100), 10))
```

Comment:
I think it might be nice to be explicit with the args here, it's just a bit more but then we don't have to remember the positional arguments if you're looking through things quickly

```suggestion
    s = channel.Slice(channel.Continuous(np.arange(num_points), start=with_offset(100), dt=10))
```